polynote-frontend/polynote/ui/layout/splitview.ts

```diff
@@ -23,7 +23,8 @@ export class SplitView extends Disposable {
                                 collapsed: !s.leftPane.collapsed
                             }
                         }
-                    })
+                    });
+                    window.dispatchEvent(new CustomEvent('resize'));
```

Comment:
What's listening for this event?

Reply:
Vega

Reply:
ah, gotcha! I see that the plots don't currently resize in `narrow_dispatcher`. I removed them because I didn't see any usages 😬  oops!